pages/Signup-login/auth.css

```diff
@@ -0,0 +1,86 @@
+
+.wrapper{
+	max-width: 40rem;
+	margin-top: 8rem;
+	margin-bottom: 2rem;
+	margin-left: 40rem;
+	width: 100%;
+	background: var(--bg-color-1);
+	padding: 3rem;
+	border-radius: 0.5rem;
+	box-shadow: 0 5px 10px rgba(0, 0, 0, 0.2);
+}
+
+.login-wrapper{
+	margin-top: 12rem;
+	margin-bottom: 6rem;
+}
+.wrapper h2 {
+	position: relative;
+	font-size: 4rem;
+	font-weight: 600;
+	color: var(--text-color-1);
+}
+
+
+.wrapper form {
+	margin-top: 1rem;
+}
+
+.wrapper form .input-box {
+	height: 3.5rem;
+	margin: 1.2rem 0;
+}
+
+form .input-box input {
+	height: 100%;
```

Comment:
Write specific CSS classes don't use descendant selectors much try to avoid them

Reply:
I realized one thing to center the Log-In Form you are using the margins and I don't understand why you did so beacuse flex is pretty much easy for this . Second thing though u used margin its still not center some What issue with the view port which can be easily avoided by using the flex It will adapt accordingly in that case